doc/sphinx/addendum/universe-polymorphism.rst

```diff
@@ -366,24 +366,32 @@ The syntax has been extended to allow users to explicitly bind names
 to universes and explicitly instantiate polymorphic definitions.
 
 .. cmd:: Universe @ident
+         Polymorphic Universe @ident
```

Comment:
Is this preferable to `{? Polymorphic} Universe @ident`?

Reply:
I think my concern was about people finding the command in the index.